board/stm32fx/stm32fx_config.h

```diff
@@ -20,6 +24,9 @@
 
 #define MAX_LED_FADE 8192U
 
+// Threshold voltage (mV) for either of the SBUs to be below before deciding harness is connected
+#define HARNESS_CONNECTED_THRESHOLD 2500U
```

Comment:
Why is this processor specific now? Doesn't look like something that should be!

Reply:
I will take a look why I needed this but think it was because of 12 bit vs 16 bit ADC and getting a lot different values from H7 ADC (and conversion formula is used for adc_get_voltage for one divider only.